aqua/yabai/files/patch-use-cc-variable.diff

```diff
@@ -0,0 +1,28 @@
+--- makefile.orig
++++ makefile
+@@ -12,6 +12,8 @@ YABAI_SRC      = ./src/manifest.m $(OSAX_SRC)
+ OSAX_PATH      = ./src/osax
+ BINS           = $(BUILD_PATH)/yabai
+ 
++CC			   ?= clang
++
+ .PHONY: all clean install sign archive man
+ 
+ all: clean-build $(BINS)
+@@ -20,8 +22,8 @@ install: BUILD_FLAGS=-std=c99 -Wall -DNDEBUG -O2 -fvisibility=hidden -mmacosx-ve
+ install: clean-build $(BINS)
+ 
+ $(OSAX_SRC): $(OSAX_PATH)/loader.m $(OSAX_PATH)/payload.m
+-	clang $(OSAX_PATH)/loader.m -shared -O2 -mmacosx-version-min=10.13 -o $(OSAX_PATH)/loader -framework Cocoa
+-	clang $(OSAX_PATH)/payload.m -DOBJC_OLD_DISPATCH_PROTOTYPES=1 -shared -fPIC -O2 -mmacosx-version-min=10.13 -o $(OSAX_PATH)/payload -framework Cocoa -framework Carbon
++	$(CC) $(OSAX_PATH)/loader.m -shared -O2 -mmacosx-version-min=10.13 -o $(OSAX_PATH)/loader -framework Cocoa
++	$(CC) $(OSAX_PATH)/payload.m -DOBJC_OLD_DISPATCH_PROTOTYPES=1 -shared -fPIC -O2 -mmacosx-version-min=10.13 -o $(OSAX_PATH)/payload -framework Cocoa -framework Carbon
```

Comment:
Don't hardcode the deployment target to be 10.13. MacPorts already sets the deployment target as an environment variable.